src/components/InlineChart.jsx

```diff
@@ -0,0 +1,178 @@
+                                height={height}
+                                width={width}
+                                padding={0}
+                            >
+                                <VictoryArea
+                                    domain={{ y: this.props.yDomain || null }}
+                                />
+                            </VictoryGroup>
+                        ));
+                        return null;
+                    });
+
+                    if (chart.mode === 'stacked') {
+                        areaCharts.push((
+                            <VictoryStack
+                                height={height}
+                                width={width}
+                                padding={0}
+                            >
+                                {areaLocal}
+                            </VictoryStack>
+                        ));
+                    } else {
+                        areaCharts = areaCharts.concat(areaLocal);
+                    }
+                    break;
+                }
+                case 'spark-bar': {
+                    const localBar = [];
+
+                    horizontal = horizontal || chart.orientation === 'left';
+
+                    Object.keys(chart.dataSetNames).map((dataSetName) => {
+                        legendItems.push({ name: dataSetName, symbol: { fill: chart.dataSetNames[dataSetName] } });
+                        localBar.push((
+                            <VictoryBar
+                                labels={d => `${config.x}:${d.x}\n${config.charts[chartIndex].y}:${d.y}`}
+                                labelComponent={
+                                    <VictoryTooltip
+                                        orientation='bottom'
+                                    />
+                                }
+                                data={dataSets[dataSetName]}
+                                color={chart.dataSetNames[dataSetName]}
+                                height={height}
+                                width={width}
+                                padding={0}
+                            />
+                        ));
+                        return null;
+                    });
+
+                    if (chart.mode === 'stacked') {
+                        barCharts.push((
+                            <VictoryStack
+                                height={height}
+                                width={width}
+                                padding={0}
+                            >
+                                {localBar}
+                            </VictoryStack>
+                        ));
+                    } else {
+                        barCharts = barCharts.concat(localBar);
+                    }
+                    break;
+                }
+                default:
+                    console.error('unsupported chart type');
```

Comment:
Use a proper logging library or a notification library such as noty for errors

Reply:
opened issue #5